ide.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://laravel-ide.com/schema/laravel-ide-v2.json",
+    "blade": {
+        "components": {
+            "fromConfig": [
+                {
+                    "file": "/config/proton.php",
```

Comment:
This should be `"/config/form-components.php"` the package doesn't publish its config as `proton.php`

Reply:
Fixed. Thanks.